client/my-sites/stats/stats-card-upsell/stats-card-upsell-wpcom.tsx

```diff
@@ -33,6 +35,10 @@ const getUpsellCopy = ( statType: string ) => {
 			return translate( 'Compare different time periods to analyze your site’s growth.' );
 		case STAT_TYPE_VIDEO_PLAYS:
 			return translate( 'Discover your most popular videos and find out how they performed.' );
+		case STATS_FEATURE_UTM_STATS:
+			return translate( 'Generate UTM parameters and track your campaign performance data.' );
```

Comment:
Raised a question to decide if we should include the Learn more link in the overlay, as we currently do on Jetpack. I think we could address this and update the links in another PR.